<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingDialog</class>
 <widget class="QWidget" name="SettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>360</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>540</width>
    <height>360</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>540</width>
    <height>360</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="settingLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>35</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Heiti Std</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Setting</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="splitLine1">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>16</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Heiti Std</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>|</string>
   </property>
  </widget>
  <widget class="QLabel" name="designedByLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Heiti Std</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Designed by</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="jyjLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>155</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Heiti Std</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Jiang Yingjin&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tryAnotherBtn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>80</y>
     <width>240</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Heiti Std</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Try another one</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="mottoCheckBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>35</y>
     <width>251</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Heiti Std</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Turn off the motto</string>
   </property>
  </widget>
  <widget class="QLabel" name="ljyLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>195</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Heiti Std</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Leikrit Li</string>
   </property>
  </widget>
  <widget class="QLabel" name="lysLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>235</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Heiti Std</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Yusheng Lin</string>
   </property>
  </widget>
  <widget class="QLabel" name="lylLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>275</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Heiti Std</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Yulong Li</string>
   </property>
  </widget>
  <widget class="QLabel" name="splitLine2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>155</y>
     <width>16</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Heiti Std</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>|</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
